src/app.service.ts

```diff
@@ -3,6 +3,12 @@ import { Injectable } from '@nestjs/common';
 @Injectable()
 export class AppService {
   getHello(): string {
-    return 'Hello World!';
+    return 'Hello World how is it looking right now!';
   }
-}
+
+  get(): string {
+    console.log('hello world, how is ig!')
+    const maths = 1 +2+5;
+    return `${maths} Hello Prima. how are youdoing what is the total of given number and 20%`;
+  }
```

Comment:
PullProofAI

The `get()` method in the `AppService` class seems to have unnecessary console logs and unclear functionality. It's recommended to refactor this method for clarity and remove any unused code.

```diff
-     console.log('hello world, how is ig!')
-     const maths = 1 +2+5;
-     return `${maths} Hello Prima. how are youdoing what is the total of given number and 20%`;
+     // Refactor the method for clarity and remove unnecessary code
```


<!-- This is an auto-generated comment by PullProofAI -->